docs/configuration/extract-codacy-logs.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+#### Helper functions ####
+
+usage()
+{
+  echo "Usage: $0 -n NAMESPACE "
+  exit 2
+}
+
+set_variable()
+{
+  local varname=$1
+  shift
+  if [ -z "${!varname}" ]; then
+    eval "$varname=\"$@\""
+  else
+    echo "Error: $varname already set"
+    usage
+  fi
+}
+
+cleanup()
+{
+	if [ -z $LOGS_DIR ]; then
```

Comment:
`-z` check if the variable is empty. Seems like this is not what you want here.
Shouldn't it be something like:
```suggestion
	if [ -d "$LOGS_DIR" ]; then
```